suite-common/message-system/src/messageSystemTypes.ts

```diff
@@ -19,12 +19,16 @@ export type MessageSystemRootState = {
 export const Feature = {
     coinjoin: 'coinjoin',
     killswitch: 'killswitch',
+    ethStake: 'eth.stake',
+    ethUnstake: 'eth.unstake',
+    ethClaim: 'eth.claim',
```

Comment:
another nitpick, feel free to ignore, but eth.claim sounds bit too generic
```suggestion
    ethStake: 'eth.staking.stake',
    ethUnstake: 'eth.staking.unstake',
    ethClaim: 'eth.staking.claim',
```

Reply:
done